### ISSUE TITLE
lensm does not display the source when building source with go 1.20

### ISSUE THREAD

**Issue by bigwhite** [loov/lensm#17]
lensm does not display the source when building source with go 1.20. lensm works ok when building source with go 1.19.

**Comment by egonelbre** [comment]
Can you try installing again. I forgot to make a new tag after making fixes for Go 1.20.

**Comment by bigwhite** [comment]
I reinstall the lensm and found that the latest lensm(v0.0.3) has fixed the problem, Thanks.  

```
$go install loov.dev/lensm@main
go: downloading loov.dev/lensm v0.0.3
go: downloading gioui.org v0.0.0-20230206180804-32c6a9b10d0b
go: downloading github.com/go-text/typesetting v0.0.0-20221214153724-0399769901d5
go: downloading github.com/benoitkugler/textlayout v0.3.0
go: downloading golang.org/x/exp v0.0.0-20221012211006-4de253d81b95
go: downloading golang.org/x/arch v0.2.0
```
